WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Package and Rate Selection/WooShippingCarrier.swift

```diff
@@ -0,0 +1,44 @@
+import UIKit
+
+/// Represents a shipping carrier in the Woo Shipping extension.
+enum WooShippingCarrier: String, Comparable, CaseIterable {
+    case ups
+    case usps
+    case dhlExpress = "dhlexpress"
+    case dhlEcommerce = "dhlecommerce"
+    case dhlEcommerceAsia = "dhlecommerceasia"
+
+    var logo: UIImage? {
+        switch self {
+        case .ups:
+            return UIImage(named: "shipping-label-ups-logo")
+        case .usps:
+            return UIImage(named: "shipping-label-usps-logo")
+        case .dhlExpress, .dhlEcommerce, .dhlEcommerceAsia:
+            return UIImage(named: "shipping-label-dhl-logo")
```

Comment:
When I was looking at Figma and dhl logo, looks like there is a new version. I was wondering if we should add a "new version" or just swap old one with new one?

New:

<img width="313" alt="Screenshot 2024-10-24 at 09 36 31" src="https://github.com/user-attachments/assets/be7760c1-9121-40a3-b80b-3aae4dc34600">

Current:

<img width="122" alt="Screenshot 2024-10-24 at 09 36 38" src="https://github.com/user-attachments/assets/a7555519-1354-450b-922c-0aaeb521d301">

Reply:
Thanks for spotting that! It seems reasonable to me to update it in place, since we wouldn't need the old one anymore. I can take care of that — I'll open a separate PR for that and test to make sure it looks ok in the old shipping label flow, too.